shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -853,8 +854,9 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         // mirrored.
         //
         // See the case above for how virtual displays are handled.
-        boolean childUsesHybridComposition = embeddedView.getContext() instanceof FlutterActivity;
-        if (childUsesHybridComposition) {
+        boolean childUsesVirtualDisplay = embeddedView.getContext() instanceof PresentationContext;
+        if (!childUsesVirtualDisplay) {
+          io.flutter.Log.d("flutter", "!childUsesVirtualDisplay");
```

Comment:
you already imported this class

Reply:
that was some left over from my debugging session :)